### PR TITLE
doc: 6725 add pcap file logging variable details v2

### PR DESCRIPTION
**Pull request by jmtaylor90**
Make sure these boxes are signed before submitting your Pull Request -- thank you.

- [x] I have read the contributing guide lines at
   https://docs.suricata.io/en/latest/devguide/contributing/contribution-process.html
- [x] I have signed the Open Information Security Foundation contribution agreement at
   https://suricata.io/about/contribution-agreement/ (note: this is only required once)
- [x] I have updated the user guide (in doc/userguide/) to reflect the changes made (if applicable)

Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket:
https://redmine.openinfosecfoundation.org/issues/6725

Describe changes:
- addressed comments in previous PR
- add pcap file name variables documentation
-

### Provide values to any of the below to override the defaults.

To use a pull request use a branch name like `pr/N` where `N` is the
pull request number.

Alternatively, `SV_BRANCH` may also be a link to an
OISF/suricata-verify pull-request.

```
SV_REPO=
SV_BRANCH=
SU_REPO=
SU_BRANCH=
LIBHTP_REPO=
LIBHTP_BRANCH=
```

### CODE REVIEW

File: doc/userguide/configuration/suricata-yaml.rst [hunk 1]
```diff
@@ -505,6 +505,27 @@ the alert.
...
+In multi mode the filename takes a few special variables:
+  - %n representing the thread number
+  - %i representing the thread id
+  - %t representing the timestamp (secs or secs.usecs based on 'ts-format')
+  
+  Example: filename: pcap.%n.%t
+
+.. note:: It is possible to use directories but the directories are not
+  created by Suricata. For example ``filename: pcaps/%n/log.%s`` will log into
+  a the pre-existing ``pcaps`` directory and per thread sub directories.
```
Comment (jufajardini) [inline doc/userguide/configuration/suricata-yaml.rst:524]: nit: a or the :P

**Review by jufajardini — COMMENTED** [review]
Looking very good to me, but spotted a sneaky typo D:

File: doc/userguide/configuration/suricata-yaml.rst (hunk 1 above)
Comment (jmtaylor90) [reply doc/userguide/configuration/suricata-yaml.rst:524]: sorry about that, fixed up

**Comment by jmtaylor90** [comment]
continued in https://github.com/OISF/suricata/pull/10294

**Pull request closed without merging**